css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 1rem;
+  left: 14rem;
```

Comment:
Woah, that's a big movement!! You want this on the right, yes? Moving it all the way from the left edge, seems like a lot of work 🤔